elokab-fa/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+
+include_directories(${QT_INCLUDES} ../library/include)
```

Comment:
This is the include dir of the library that you link to anyway?

Then better make it a public include dir of the library and remove the include directories call here